### ISSUE TITLE
Getting "file doesn't exist!" error for all files, fixed by restart

### ISSUE THREAD

**Issue by yozlet** [RandomFractals/vscode-data-preview#69]
Hi! This extension looks great, and I'm eager to use it. 

I couldn't seem to get it to work after installing, **including after a window reload**. For every file I tried to preview, I got a "file doesn't exist!" error, and there was no further information in the logs. It was only after fully shutting down VSCode and restarting it that the extension was able to load files.

So, listing out the individual problems worth fixing here, which may need to be broken out into separate issues. (They're not criticisms, just things that would help the new user experience):
1. Extension shouldn't need a full VSCode restart to start working; a window reload at most.
2. The error message was incorrect: the file _did_ exist. What was actually happening here, and why wasn't a more helpful error given?
3. No log information to help debug.

I'm running on MacOS 10.14.5, VSCode 1.35.1, Data Preview 1.21.0.

Anyway, I have the extension running now, and it's great! It's particularly great that the improvements are coming so quickly. I'm looking forward to filing more improvement suggestions. 😉

**Comment by RandomFractals** [comment]
yeah, there are a few things going there. I'll parse through it tomorrow.

wrapping up optimized arrow data loading release today.

I appreciate your thorough feedback notes and running on v's list

**Comment by RandomFractals** [comment]
@yozlet this v1.x ext. requires full vscode reload for now on new install or latest release update since it's data view is still changing daily and your 'windows', or data preview panels to be more precise will not restore correctly on new ext. v. update. 

Please use Save config option between vscode reloads to save your data view changes.

I created new ticket to improve on file not found, etc. error messages display.
